src/lib/customTraceType.js

```diff
@@ -1,13 +1,15 @@
 export function plotlyTraceToCustomTrace(trace) {
   if (
     trace.type === 'scatter' &&
+    trace.fill &&
     ['tozeroy', 'tozerox', 'tonexty', 'tonextx', 'toself', 'tonext'].includes(
       trace.fill
     )
   ) {
     return 'area';
   } else if (
     trace.type === 'scatter' &&
+    trace.mode &&
     (trace.mode === 'lines' || trace.mode === 'lines+markers')
```

Comment:
are `&& trace.fill` and `&& trace.mode` actually necessary here? falsey values will fail the following clauses anyway.

Reply:
yeah..you're right, removing